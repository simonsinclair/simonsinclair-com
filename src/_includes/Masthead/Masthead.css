.masthead {
  align-items: center;
  display: flex;
  justify-content: space-between;
  padding-top: 1.5rem;
}

.masthead__logo {
  font-style: normal;
  font-weight: 700;
}
