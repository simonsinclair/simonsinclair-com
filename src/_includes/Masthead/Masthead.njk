<header class="masthead container">
  <strong class="masthead__logo">
    <a href="/">Simon Sinclair</a>
  </strong>
  {% include "NavBar/NavBar.njk" %}
</header>
