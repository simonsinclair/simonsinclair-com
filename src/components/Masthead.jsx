import Link from 'next/link';

import NavBar from './NavBar';

const Masthead = () => (
  <header>
    <h1 className="logotype"><Link href="/"><a>Simon Sinclair</a></Link></h1>
    <NavBar />

    <style jsx>
      {`
        .logotype {}
      `}
    </style>
  </header>
);

export default Masthead;
