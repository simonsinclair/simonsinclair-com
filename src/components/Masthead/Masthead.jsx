import Link from 'next/link';
import styled from 'styled-components';

import NavBar from '../NavBar/NavBar';

const Header = styled.header`
  align-items: center;
  display: flex;
  justify-content: space-between;
  padding-top: 1.5rem;

  .logo {
    font-style: normal;
    font-weight: 700;
  }
`;

const Masthead = () => (
  <Header className="container">
    <strong className="logo">
      <Link href="/">
        <a>Simon Sinclair</a>
      </Link>
    </strong>

    <NavBar />
  </Header>
);

export default Masthead;
