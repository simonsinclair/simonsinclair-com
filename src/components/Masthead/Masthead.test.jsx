import { render } from '@testing-library/react';

import Masthead from './Masthead';

describe('<Masthead /> is a component that', () => {
  it('renders and matches its snapshot', () => {
    const { container } = render(<Masthead />);
    expect(container).toMatchSnapshot();
  });
});
