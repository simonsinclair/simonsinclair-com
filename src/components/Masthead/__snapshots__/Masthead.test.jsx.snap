// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Masthead /> is a component that renders and matches its snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding-top: 1.5rem;
}

.c0 .logo {
  font-style: normal;
  font-weight: 700;
}

<div>
  <header
    class="container c0"
  >
    <strong
      class="logo"
    >
      <a
        href="/"
      >
        Simon Sinclair
      </a>
    </strong>
    <nav
      class="nav-bar"
    >
      <ul
        class="nav-list nav-list--inline"
      >
        <li>
          <a
            class=""
            href="/"
          >
            Home
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
